Changed the alert rule of TiKV_thread_apply_worker_cpu_seconds (#7068)
pingcap/docs: alert-rules.md

* Alert rule:

    `max(rate(tikv_thread_cpu_seconds_total{name=~"apply_.*"}[1m])) by (instance) > 0.9`

* Description:

>     The pressure on the apply Raft log thread is too high, and has approached or exceeded the limit of the `apply` thread. It is often caused by a burst of writes.

Reviewer: ```suggestion
    The apply Raft log thread suffers from high pressure and has approached or exceeded its limit. This is often caused by a burst of writes.
```
The subject of 'has approached or exceeded the limit of the `apply` thread' is 'The pressure', which is inappropriate.

* Alert rule:

>     `max(rate(tikv_thread_cpu_seconds_total{name=~"apply_.*"}[1m])) by (instance) > 0.9`

Reviewer: 告警描述中文有优化，看看英文要不要跟中文对齐.

Author: 好的。已对齐。
